webpack.config.babel.js

```diff
@@ -246,7 +197,7 @@ const configDist = {
     library: 'Onfido',
     libraryTarget: 'umd',
     path: `${__dirname}/dist`,
-    publicPath: CONFIG.PUBLIC_PATH,
+    publicPath: CONFIG.PUBLIC_PATH ?? `${CONFIG.PUBLIC_PATH_PREFIX}/${packageJson.version}/`,
```

Comment:
Looks like `PUBLIC_PATH` is defined for all environments as `/` so `PUBLIC_PATH_PREFIX` will never be returned.